lib/db/query.js

```diff
@@ -107,6 +117,18 @@ async function addNewTestAssertions(testId, assertions) {
   return true;
 }
 
+async function editTestAssertions(testId, assertions) {
+  const query = `
+    DELETE FROM assertions WHERE test_id = ${testId}
```

Comment:
I'm concerned about deleting all of these assertions, since old test run data relies on these rows existing. What do you think?